custom_msg/info.json

```
@@ -0,0 +1,14 @@
+{
+    "author": [
+        "Issy"
+    ],
+    "short": "",
+    "description": "",

```

```suggestion
    "short": "Send messages as the bot",
    "description": "Allows moderators to send messages as the bot, also allowing modification of existing messages sent from the cog",

```

Cog short & description.